prow/cluster/jobs/istio/proxy/istio.proxy.master.yaml

```diff
@@ -50,10 +50,29 @@ presubmits:
     context: prow/proxy-presubmit.sh
     branches: *branch_spec
     always_run: true
+    decorate: true
+    path_alias: istio.io/proxy
+    timeout: 60m
+    extra_refs:
+    - org: istio.io
```

Comment:
This isn't a valid GitHub org: https://github.com/istio.io/
I think you mean:
```yaml
- org: istio
  repo: istio
  base_ref: master
  path_alias: istio.io/istio
```

Reply:
Done. But also, it did not fail when I was testing it.

Reply:
Tested with `phaino` right? `phaino` doesn't respect git refs properly right now. I thought we had an open issue for this, but I couldn't find one. Opened here: https://github.com/kubernetes/test-infra/issues/13452

Reply:
Yes, i tested it with phaino